[2022-05-30 18:52:48,567] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: data_ingestion_gcs_dag.bigquery_external_table_task manual__2022-05-30T18:51:08.693839+00:00 [queued]>
[2022-05-30 18:52:48,584] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: data_ingestion_gcs_dag.bigquery_external_table_task manual__2022-05-30T18:51:08.693839+00:00 [queued]>
[2022-05-30 18:52:48,586] {taskinstance.py:1357} INFO - 
--------------------------------------------------------------------------------
[2022-05-30 18:52:48,587] {taskinstance.py:1358} INFO - Starting attempt 1 of 2
[2022-05-30 18:52:48,588] {taskinstance.py:1359} INFO - 
--------------------------------------------------------------------------------
[2022-05-30 18:52:48,606] {taskinstance.py:1378} INFO - Executing <Task(BigQueryCreateExternalTableOperator): bigquery_external_table_task> on 2022-05-30 18:51:08.693839+00:00
[2022-05-30 18:52:48,617] {standard_task_runner.py:52} INFO - Started process 217 to run task
[2022-05-30 18:52:48,622] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'data_ingestion_gcs_dag', 'bigquery_external_table_task', 'manual__2022-05-30T18:51:08.693839+00:00', '--job-id', '32', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_gcs_dag.py', '--cfg-path', '/tmp/tmp8hjp42ky', '--error-file', '/tmp/tmpuy129_kf']
[2022-05-30 18:52:48,625] {standard_task_runner.py:80} INFO - Job 32: Subtask bigquery_external_table_task
[2022-05-30 18:52:48,695] {task_command.py:370} INFO - Running <TaskInstance: data_ingestion_gcs_dag.bigquery_external_table_task manual__2022-05-30T18:51:08.693839+00:00 [running]> on host e083cbf9476b
[2022-05-30 18:52:48,762] {taskinstance.py:1572} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=data_ingestion_gcs_dag
AIRFLOW_CTX_TASK_ID=bigquery_external_table_task
AIRFLOW_CTX_EXECUTION_DATE=2022-05-30T18:51:08.693839+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-05-30T18:51:08.693839+00:00
[2022-05-30 18:52:48,766] {crypto.py:82} WARNING - empty cryptography key - values will not be stored encrypted.
[2022-05-30 18:52:48,767] {base.py:68} INFO - Using connection ID 'google_cloud_default' for task execution.
[2022-05-30 18:52:49,830] {taskinstance.py:1401} INFO - Marking task as SUCCESS. dag_id=data_ingestion_gcs_dag, task_id=bigquery_external_table_task, execution_date=20220530T185108, start_date=20220530T185248, end_date=20220530T185249
[2022-05-30 18:52:49,891] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-05-30 18:52:49,933] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
